Add NollySenti Bitext Mining (MINERS)

gentaiscool opened the pull request:
NollySenti is Nollywood movie reviews for five languages widely spoken in Nigeria (English, Hausa, Igbo, Nigerian-Pidgin, and Yoruba.

## Checklist
<!-- Please do not delete this -->

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 


### Adding datasets checklist
<!-- see also https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md -->

**Reason for dataset addition**: ... <!-- Add reason for adding dataset here. E.g. it covers task/language/domain previously not covered -->

- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`.

gentaiscool commented:
@KennethEnevoldsen I created a new PR for NollySenti.

gentaiscool commented:
I have completed the checklist @KennethEnevoldsen.

KennethEnevoldsen commented on mteb/tasks/BitextMining/multilingual/NollySentiBitextMining.py:
```diff
@@ -0,0 +1,49 @@
+        }
+        """,
+        n_samples={"train": 1640},
+        avg_character_length={"train": 4.46},
```
a bit short?

gentaiscool replied:
I updated the avg character length per sample (previously I put the avg char per word)

KennethEnevoldsen reviewed (commented):
Will you add points and push models results as well?

gentaiscool commented:
Thank you for the review. I addressed them. I updated the avg number of character per sample, added the results, and points. The new languages are `hau-Latn`, `ibo-Latn`, `pcm-Latn`, `yor-Latn`, so the points are 2 + 4 x 4 = 18. Plus, I added the point for the reviewer. @KennethEnevoldsen
